docs/source/agents_list.rst

```diff
@@ -1,10 +0,0 @@
-..
-  Copyright (c) Facebook, Inc. and its affiliates.
-  This source code is licensed under the MIT license found in the
-  LICENSE file in the root directory of this source tree.
-
-ParlAI Agents
```

Comment:
This was actually something that Jason wanted, so that there was a top-level thing for agents

Reply:
Ugh I was trying to fight it to have like, subcontents while still having separate pages.